System.php

```diff
@@ -480,7 +480,7 @@ public static function tmpdir()
         if ($var = isset($_ENV['TMPDIR']) ? $_ENV['TMPDIR'] : getenv('TMPDIR')) {
             return $var;
         }
-        return realpath('/tmp');
+        return realpath(version_compare(PHP_VERSION, '5.2.1', '>=') ? sys_get_temp_dir() : '/tmp');
```

Comment:
could this be simply a `function_exists` check, what do you think?

Reply:
Yes, `function_exists` is probably better, I updated the PR. Can you merge?